UnitTests/UnitTests.csproj

```diff
@@ -14,6 +14,7 @@
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
so we need this one at all?

Reply:
This is a "leftover" from when .net 4.6.1 was a target platform here.
If you plan on leaving the .net 4.6.1 in it's own UnitTest project, then this whole ItemGroup can actually go.
If you might want to merge all the test platforms into one test project, then having it in here will mean it's prepared for that time.

If you want, I can revert adding this line. Or remove the whole ItemGroup all together.

Reply:
No, i don't think i'm planning to keep `net461` target here. Guess it was added in effort to make single test suite, but it appeared to be pain in the ass. Too many conditional complication stuff.

Reply:
Okay, I've removed the ItemGroup to clean up the file.